webconsole/v1/register.go

```diff
@@ -0,0 +1,31 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	GroupName = "webconsole.config.openshift.io"
```

Comment:
the group follows the pattern of `kubeproxy.config.k8s.io` here https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/apis/kubeproxyconfig/register.go#L30

Reply:
I'm fine with this.